### PR TITLE
QOLSVC-978 update DEV environment to CKAN 2.10 and Solr 8

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

**Review by RossWebsterWork — APPROVED** [review]

File: test/features/environment.py [hunk 1]
```diff
@@ -118,4 +118,5 @@ def before_scenario(context, scenario):
 
 
 def after_scenario(context, scenario):
+    os.system("ckan_cli jobs clear")
```
Comment (duttonw) [inline test/features/environment.py:121]: nice

File: vars/shared-OpenData.var.yml [hunk 2]
```diff
@@ -49,13 +49,13 @@ extensions:
...
       version: "v1.2.0"
 
-    CKANExtScheming: &CKANExtScheming
-      name: "ckanext-scheming-{{ Environment }}"
-      shortname: "ckanext-scheming"
-      description: "CKAN Extension for Scheming"
-      type: "git"
-      url: "https://github.com/ckan/ckanext-scheming.git"
-      version: "release-3.0.0"
+    #~ CKANExtScheming: &CKANExtScheming
```
Comment (duttonw) [inline vars/shared-OpenData.var.yml:52]: why was this removed?

Comment (ThrawnCA) [reply vars/shared-OpenData.var.yml:52]: It's already a transitive dependency of ckanext-validation, and for some reason adding it directly causes a clash on the test containers where it unsuccessfully tries to replace the distribution version of `six`. I've tried a bunch of ways to tell it to Not Do That, but none seemed to work. Letting it be installed transitively worked.
